recipes/gds-root-extensions/meta.yaml

```diff
@@ -0,0 +1,137 @@
+{% set name = "gds-root-extensions" %}
+{% set version = "2.19.4" %}
+
+# dependencies
+{% set gds_base_version = "2.19.5" %}
+{% set gds_lsmp_version = "2.19.7" %}
+{% set gds_frameio_version = "2.19.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://software.igwn.org/sources/source/{{ name }}-{{ version }}.tar.gz
+  sha256: 275c43b5e6ad61357e1a1053210fe05522cfae0fa77213fa72a13ae3948b0995
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config
+  host:
+    - gds-base >={{ gds_base_version }}
+    - gds-base-crtools >={{ gds_base_version }}
+    - gds-base-gdstrig >={{ gds_base_version }}
+    - gds-frameio-base >={{ gds_frameio_version }}
+    - gds-lsmp >={{ gds_lsmp_version }}
+    - root_base
+    - root-binaries
```

Comment:
I guess you're already aware but you'll have to fix this once you have a migration PR to ROOT 6.24 (sorry for the trouble, the `root-binaries` split was a bad idea).

Reply:
Aye, no problem, thanks for the reminder.